content/docs/admin/mail-config.md

```diff
@@ -0,0 +1,92 @@
++++
+title = "Mail Configuration"
+description = "Configuring Bookstack to send mail"
+date = "2017-08-08"
+type = "admin-docs"
+slug = "mail-config"
++++
+
+This document has been written by referencing the file at `config/mail.php`.
+
+Bookstack supports the following mechanisms for sending mail:
+
+1. SMTP;
+2. mail (php sendmail() function);
+3. sendmail (Linux sendmail);
+4. Mailgun (Rackspace Mailgun);
+5. Mandrill;
+6. Amazon SES ('ses'); or
+7. Log
+
```

Comment:
@ssddanbrown Thanks for the update on the other pull request - Are these dependencies components within the Laravel framework, or operating system dependencies? I've got a containerised deployment of Bookstack running with the Mailgun API credentials at the moment, and it seems to work okay.

I'd still like to add a footnote about these dependencies - It will also prevent people from filing erroneous issues claiming that the API-based mail sending mechanisms are broken.